tests/load/sources/sql_database/test_sql_database_source_all_destinations.py

```diff
@@ -155,6 +156,9 @@ def test_load_sql_table_incremental(
     """
     os.environ["SOURCES__SQL_DATABASE__CHAT_MESSAGE__INCREMENTAL__CURSOR_PATH"] = "updated_at"
 
+    if not IS_SQL_ALCHEMY_20 and backend == "connectorx":
```

Comment:
we can probably get these to work again on sqlalchemy 1.4 if we don't use an date at but an int column for incremental